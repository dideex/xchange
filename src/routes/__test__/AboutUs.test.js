import React from 'react'
import Component from '../AboutUs'
import {shallow} from 'enzyme'

describe('Routes about us', () => {
  it('Markup', () => {
    const wrapper = shallow(<Component />)

    expect(wrapper.html()).toMatchSnapshot()
  })
})