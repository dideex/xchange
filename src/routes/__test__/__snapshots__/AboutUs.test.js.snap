// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes about us Markup 1`] = `"<section class=\\"css-9vgqxq\\"><main class=\\"css-15rhx1u\\"><div class=\\"css-1xiihgh\\"><h2 class=\\"css-x184e\\"><span>О нас</span></h2><p class=\\"css-n13e7f\\"><span>Наш сервис помогает производить обмен Биткоин безопасно и быстро.</span></p><p class=\\"css-n13e7f\\"><span>Все операции по конвертации валют занимают не более 15 минут. Компетентная онлайн-поддержка с удовольствием проконсультирует Вас на всех этапах работы с нашим сервисом. Выгодные курсы и качественное обслуживание - залог работы с клиентами.</span></p></div></main></section><section class=\\"css-1hhczj\\"><svg class=\\"main__bottom-background\\" style=\\"width:60%\\" viewBox=\\"0 0 857 441\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path style=\\"fill:#40c0cb\\" d=\\"M0.5 -0.00012207C38.9999 261.5 357 375.5 856.5 440.5V-0.00012207H0.5Z\\"></path></svg><svg class=\\"main__bottom-bigcoin\\" style=\\"width:166px\\" version=\\"1.1\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><title>Bitcoin</title><circle style=\\"fill:#edda48\\" cx=\\"199.5\\" cy=\\"200.5\\" r=\\"199.5\\"></circle><circle style=\\"fill:#e2b255\\" cx=\\"199.5\\" cy=\\"200.5\\" r=\\"168.3\\"></circle><path style=\\"fill:#edda48\\" d=\\"M262.5,193.9c47-38.9-0.7-74.8-36.9-73.2v-38h-26.1v38.5H186V82.8h-24.2v38h-45.1v22.5h28.6v119.9h-28.6v23.1 h45.1v37.4h23.7v-37.9h14.1v37.9h23.9v-37.9C279,284.1,319.4,238.7,262.5,193.9z M186,145.8h34.4c13.2,1.4,26.2,27.8,0,42.1H186 V145.8z M225.1,261.9H186v-49.6h39.1C252.9,220.8,254.8,249.2,225.1,261.9z\\"></path></svg><svg class=\\"main__bottom-smallcoin\\" style=\\"width:84px\\" version=\\"1.1\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><title>Bitcoin</title><circle style=\\"fill:#edda48\\" cx=\\"199.5\\" cy=\\"200.5\\" r=\\"199.5\\"></circle><circle style=\\"fill:#e2b255\\" cx=\\"199.5\\" cy=\\"200.5\\" r=\\"168.3\\"></circle><path style=\\"fill:#edda48\\" d=\\"M262.5,193.9c47-38.9-0.7-74.8-36.9-73.2v-38h-26.1v38.5H186V82.8h-24.2v38h-45.1v22.5h28.6v119.9h-28.6v23.1 h45.1v37.4h23.7v-37.9h14.1v37.9h23.9v-37.9C279,284.1,319.4,238.7,262.5,193.9z M186,145.8h34.4c13.2,1.4,26.2,27.8,0,42.1H186 V145.8z M225.1,261.9H186v-49.6h39.1C252.9,220.8,254.8,249.2,225.1,261.9z\\"></path></svg><svg class=\\"main__bottom-dollar\\" style=\\"width:204px\\" viewBox=\\"0 0 243 198\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path fill=\\"#A5C957\\" d=\\"M195.543 0.5L243 96.763L57.7707 197.43L-0.00012207 92.8481L195.543 0.5Z\\"></path><path fill=\\"#8EB85B\\" d=\\"M195.543 0.5L-0.00012207 92.8481L57.7707 197.5L243 96.8329L195.543 0.5ZM62.3004 178.905L17.0036 96.9727L188.713 17.6274L221.676 92.149L62.3004 178.905Z\\"></path><path fill=\\"#8EB85B\\" d=\\"M90.8026 146.398L87.597 140.036C96.1685 133.814 97.2138 125.565 93.381 119.134C89.4785 112.492 83.4157 111.164 74.0079 112.772C73.9382 112.772 73.7989 112.772 73.7292 112.842L65.6454 96.833C69.1995 95.0154 72.4051 94.6658 74.1473 94.5959L72.8929 88.8635C70.5235 88.9334 67.0392 89.7024 63.067 91.7996L60.4886 86.7662L56.3073 88.3741L59.2342 94.3163C52.4745 99.6293 50.6626 107.249 54.4258 113.541C57.8404 119.343 63.8336 120.741 71.6385 119.553L80.4888 137.589C77.1439 139.057 73.5898 139.686 70.7326 139.686L72.196 145.419C74.9835 145.559 79.1648 144.51 82.9976 142.692L85.6457 148.145L90.8026 146.398ZM87.7363 123.118C90.1057 127.103 89.1998 131.507 85.0185 134.933L76.7954 118.854C82.4401 118.295 85.576 119.483 87.7363 123.118ZM59.9311 109.346C58.3979 106.76 58.3282 102.845 61.8126 99.4894L68.5723 113.331C64.3911 113.471 61.7429 112.422 59.9311 109.346Z\\"></path><path fill=\\"#8EB85B\\" d=\\"M192.407 99.699L189.201 93.3374C197.773 87.1156 198.818 78.8665 194.985 72.4349C191.083 65.7937 185.02 64.4655 175.612 66.0733C175.542 66.0733 175.403 66.0733 175.333 66.1433L167.25 50.2043C170.804 48.3867 174.009 48.0371 175.752 47.9672L174.497 42.2348C172.128 42.3047 168.643 43.0737 164.671 45.1709L162.093 40.1376L157.912 41.7455L160.838 47.6876C154.079 53.0006 152.267 60.6205 156.03 66.9122C159.445 72.7146 165.438 74.1127 173.243 72.9243L182.093 90.9605C178.748 92.4286 175.194 93.0577 172.337 93.0577L173.8 98.7902C176.588 98.93 180.769 97.8814 184.602 96.0638L187.25 101.517L192.407 99.699ZM189.341 76.4197C191.71 80.4044 190.804 84.8086 186.623 88.2341L178.469 72.0854C184.044 71.5961 187.18 72.8544 189.341 76.4197ZM161.535 62.7178C160.002 60.1312 159.932 56.2164 163.417 52.8608L170.177 66.7025C165.995 66.8423 163.347 65.7937 161.535 62.7178Z\\"></path></svg><svg style=\\"width:186px\\" class=\\"main__bottom-card\\" viewBox=\\"0 0 215 174\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M164.004 169.243L11.1657 114.674C5.01935 112.48 1.80568 105.63 4.03531 99.3849L35.3055 11.8026C37.5351 5.55779 44.3604 2.29215 50.5067 4.48661L203.345 59.0556C209.543 61.2684 212.705 68.1 210.494 74.2932L179.205 161.927C176.976 168.172 170.15 171.438 164.004 169.243Z\\" fill=\\"white\\"></path><path d=\\"M50.6775 86.0123L30.1898 78.6974C29.3191 78.3865 28.8928 77.4779 29.206 76.6005L31.6015 69.8912C31.9147 69.0138 32.8201 68.5806 33.6909 68.8915L54.1786 76.2064C55.0493 76.5173 55.4756 77.4259 55.1623 78.3033L52.7669 85.0126C52.4536 85.89 51.5482 86.3232 50.6775 86.0123Z\\" fill=\\"#BABABA\\"></path><path d=\\"M45.5179 100.463L25.0302 93.1482C24.1595 92.8373 23.7332 91.9287 24.0465 91.0513L26.4419 84.342C26.7552 83.4646 27.6606 83.0314 28.5313 83.3423L49.019 90.6572C49.8898 90.9681 50.3161 91.8767 50.0028 92.7541L47.6073 99.4634C47.2941 100.341 46.3887 100.774 45.5179 100.463Z\\" fill=\\"#BABABA\\"></path><path d=\\"M79.6163 96.3446L59.1286 89.0297C58.2579 88.7188 57.8316 87.8102 58.1449 86.9328L60.5403 80.2235C60.8536 79.3461 61.759 78.9129 62.6297 79.2238L83.1174 86.5387C83.9881 86.8496 84.4144 87.7582 84.1012 88.6356L81.7057 95.3449C81.3925 96.2223 80.4871 96.6555 79.6163 96.3446Z\\" fill=\\"#BABABA\\"></path><path d=\\"M108.555 106.677L88.0676 99.362C87.1969 99.0511 86.7706 98.1424 87.0838 97.2651L89.4793 90.5557C89.7926 89.6784 90.698 89.2452 91.5687 89.5561L112.056 96.8709C112.927 97.1818 113.353 98.0905 113.04 98.9679L110.645 105.677C110.331 106.555 109.426 106.988 108.555 106.677Z\\" fill=\\"#BABABA\\"></path><path d=\\"M137.494 117.009L117.006 109.694C116.136 109.383 115.709 108.475 116.023 107.597L118.418 100.888C118.731 100.011 119.637 99.5775 120.508 99.8883L140.995 107.203C141.866 107.514 142.292 108.423 141.979 109.3L139.584 116.009C139.27 116.887 138.365 117.32 137.494 117.009Z\\" fill=\\"#BABABA\\"></path><path d=\\"M132.335 131.46L111.847 124.145C110.976 123.834 110.55 122.926 110.863 122.048L113.259 115.339C113.572 114.461 114.477 114.028 115.348 114.339L135.836 121.654C136.706 121.965 137.133 122.874 136.819 123.751L134.424 130.46C134.111 131.338 133.205 131.771 132.335 131.46Z\\" fill=\\"#BABABA\\"></path><path d=\\"M188.742 78.2815L104.999 48.3819C103.513 47.8516 102.726 46.1741 103.261 44.6774L104.827 40.2905C105.361 38.7938 107.033 37.9941 108.518 38.5244L192.262 68.424C193.747 68.9543 194.534 70.6318 194 72.1285L192.434 76.5154C191.848 77.9938 190.176 78.7935 188.742 78.2815Z\\" fill=\\"#BABABA\\"></path><path d=\\"M59.6818 69.8503L41.6526 63.4132C38.8356 62.4074 37.3927 59.3319 38.4062 56.4933L43.6394 41.8361C44.6528 38.9975 47.7173 37.5313 50.5343 38.5371L68.6147 44.9925C71.4318 45.9983 72.8747 49.0738 71.8612 51.9123L66.6096 66.6212C65.5633 69.3899 62.4989 70.8561 59.6818 69.8503Z\\" fill=\\"#DCDC9A\\"></path></svg></section><section class=\\"css-uvb5el\\"><h2 class=\\"css-x184e\\"><span>Наша команда</span></h2><div class=\\"css-rdbcp7\\"><div class=\\"slick-slider slick-initialized\\" dir=\\"ltr\\"><svg style=\\"display:block;fill:#191716\\" class=\\"chevron__left\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#191716\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg><div class=\\"slick-list\\"><div class=\\"slick-track\\" style=\\"width:500%;left:-100%\\"><div data-index=\\"-2\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"-1\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"0\\" class=\\"slick-slide slick-active slick-current\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline:none;width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff1.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Абакумов М.А.</h3><p>Генеральный директор</p></article></div></div><div data-index=\\"1\\" class=\\"slick-slide slick-active\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline:none;width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff2.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"2\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline:none;width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"3\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline:none;width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"4\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff1.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Абакумов М.А.</h3><p>Генеральный директор</p></article></div></div><div data-index=\\"5\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff2.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"6\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"7\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width:10%\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"/><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div></div></div><svg style=\\"display:block;fill:#191716\\" class=\\"chevron__right\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#191716\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div></section>"`;
